tests/shell/Makefile

```diff
@@ -6,10 +6,17 @@ USEMODULE += shell
 USEMODULE += shell_commands
 USEMODULE += ps
 
+# for z1, socat doesn't work (unknown reason)
+ifeq (z1, $(BOARD))
+  RIOT_TERMINAL ?= pyterm
+endif
```

Comment:
I agree with this workaround not worth spending time investigating an old non available `BOARD`